src/SqlProviderBase.ts

```diff
@@ -253,7 +260,8 @@ export abstract class SqlProviderBase extends NoSqlProvider.DbProvider {
                             fieldList.push('nsp_pk TEXT PRIMARY KEY');
                             fieldList.push('nsp_data TEXT');
 
-                            const nonMultiIndexes = _.filter(storeSchema.indexes || [], index => !index.multiEntry);
+                            const nonMultiIndexes = _.filter(storeSchema.indexes, index =>
```

Comment:
nonMultiIndexes isn't quite the proper name anymore since this is 
!indexSchema.multiEntry && (!indexSchema.fullText || !supportsFTS3)